dev-cpp/fast_double_parser/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<upstream>
```

Comment:
Can you add yourself as maintainer?
You will have to create an account on bugs.gentoo.org as well with the same email.
This will allow you to receive notification emails when bugs are reported for this package.
Alternatively, you could put @vitaly-zdanevich as maintainer instead if he agrees to it.

Reply:
Ok, I will do it.